src/main/java/seedu/address/model/person/Id.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's unique id in the address book, smallest id is 1.
+ * Guarantees: immutable; is valid as declared in {@link #isValidId(int)}
+ */
+public class Id {
+
+    public static final String MESSAGE_CONSTRAINTS = "Ids should only be any integer value larger than 0";
```

Comment:
maybe in the string Id can be written as IDs

Reply:
Standardised as `id`.